src/applications/mhv-medications/util/helpers.js

```diff
@@ -427,6 +427,7 @@ export const sanitizeKramesHtmlStr = htmlString => {
       return word;
     });
     h2.textContent = words.join(' ');
+    h2.setAttribute('id', h2.textContent);
```

Comment:
are you meaning to set the ID as the text content here? Seems like it might be a big ID lol

Reply:
yes, the issue is that the navigation component wasn't picking up the inner text of the h2, but it did pick up the id. Given that the function used a map, I would have to break it down and make a case for each h2 and give it a more specific id related to the title/content.

I didn't see any issues with the long id other than it being long. I think if it becomes a problem or if ucd wants to manipulate each title maybe we can break down the function, until then idt its necessary to complicate the function...yet. Unless you disagree I'm open to being won over 😇